doc/getting_started/install.rst

```diff
@@ -25,91 +28,136 @@ The installed python is a full version of python. If your specialized
+and other tools used in the development process.
 
-There are couple of options for setting up a python environment on windows:
+Javascript Environment
+----------------------
 
-  - `python.org <https://www.python.org/>`_, and
-  - `Anaconda <https://www.anaconda.com/distribution/>`_.
+If you are planning to develop the webview (client), you will need to install
+a Javascript environment.
 
-With most pypi packages now bundled with wheels, it is now easy to set up a
-development environment using the official python package.  Similarly,
-anaconda provides binaries for all the refl1d dependencies.
+* `Node.js <https://nodejs.org/en/download/>`_ can be installed from the website, or using conda::
 
-You will need a C/C++ compiler.  If you already have Microsoft Visual C
-installed you are done. If not, you can use the MinGW compiler that is supplied
-with your python environment or download your own.  You can set MinGW
-as the default compiler by creating the file *Lib\distutils\\distutils.cfg*
-in your python directory (e.g., *C:\\Python3.8*) with the following content::
+    conda install -c conda-forge nodejs
 
-    [build]
-    compiler=mingw32
+* `Bun <https://bun.sh/>`_ is a fast-performing drop-in replacement for npm, and is available on all platforms. 
 
-Once the python is prepared, you can install the periodic table and bumps
-package using the Windows console.
+Similar to the Python environment, you may want to install and link the ``bumps-webview-client`` in your Refl1d Javascript environment::
 
-Linux
------
+    cd /path/to/bumps/bumps/webview/client
+    npm install # or bun install
+    npm link    # or bun link
 
-Linux distributions will provide the base required packages.  You
-will need to refer to your distribution documentation for details.
+    cd /path/to/refl1d/refl1d/webview/client
+    npm install                     # or bun install
+    npm link bumps-webview-client   # or bun link bumps-webview-client
 
-On debian/ubuntu, the command will be something like::
+To build the client, run::
 
-    sudo apt-get install python3-{matplotlib,numpy,scipy,wxgtk4.0,pyparsing,setuptools}
+    cd /path/to/refl1d/refl1d/webview/client
+    npm run build   # or bun build
 
-For development you also want nose and sphinx::
+If you are developing the client, you can run the client in development mode.
+In this mode, any changes to client code are immediately reflected in a connected running client::
 
-    sudo apt-get install python-{nose,sphinx}
+    npm run dev   # or bun run dev
 
-Latex is needed to build the pdf documentation.
+This starts the client and shows the URL to connect to in the terminal (typically http://localhost:5173).
 
-OS/X
-----
+Now, you can start the Python webview server with::
 
-Similar to windows, you can install the official python distribution or
-use Anaconda.  You will need to install the Xcode command line utilities
-to get the compiler.
+    refl1d-webview --headless --port 8080
```

Comment:
I would not recommend `--headless` as the default... if they leave that off, it will open a browser for them automatically, using the correct localhost url.

Reply:
My understanding was that going to `localhost:8080` took you to the version that uses the pre-compiled JS, rather than the local Vue/TS which is getting served at `localhost:5173/?server=localhost:8080`

This section is meant to be "if you are also developing the javascript environment. 

I can add an additional reference without `--headless` in the section above